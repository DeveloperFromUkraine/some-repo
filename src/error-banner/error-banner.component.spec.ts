import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ErrorBannerComponent } from "../index";
import { ComponentTest } from "../../test/test-bed/component";

describe('Error Banner', () => {
    let fixture: ComponentFixture<ErrorBannerComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ErrorBannerComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ErrorBannerComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});