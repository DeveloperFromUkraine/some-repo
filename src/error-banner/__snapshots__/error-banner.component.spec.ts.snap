// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBannerComponent should match snapshot 1`] = `
<ign-error-banner>
  <p>
    
    
    
  
  </p>
</ign-error-banner>
`;
